src/discussions/discussions-home/DiscussionsHome.jsx

```diff
@@ -72,7 +70,7 @@ const DiscussionsHome = () => {
       }}
       >
         {!enableInContextSidebar && (
-          <Header courseOrg={org} courseNumber={courseNumber} courseTitle={courseTitle} />
+          <Header />
```

Comment:
Can't we just have our Header component show any text we pass to it? so we don't need to delete the data showing in the header?

Reply:
We don’t have a specific header for this MFE; we use a unique header for all MFEs. Here, they use the learning header, which has differences compared to the main header.